package.json

```diff
@@ -177,5 +177,13 @@
     "vinyl-buffer": "1.0.1",
     "vinyl-source-stream": "2.0.0",
     "watchify": "3.11.1"
+  },
+  "filesize": {
+    "track": [
+      "dist/*.js",
+      "dist/v0/*-?.?.js",
+      "dist/*.mjs",
+      "dist/v0/*-?.?.mjs"
+    ]
```

Comment:
Can `@ampproject/filesize` read config data from a standalone file (something like `.filesizerc.json`, similar to [`.renovaterc.json`](https://docs.renovatebot.com/configuration-options/)) as opposed the the project's main `package.json`? If so, or if it's possible to make it so, I'd encourage it, since this list may grow in future, and it would be nice to track changes to it separately.

Reply:
Adding onto Raghu's comment, splitting it out would allow us to make `wg-performance` an OWNER of the filesize file, whereas right now `wg-infra` owns `package.json`

Reply:
We can add complexity into `filesize` for this case, but this preference was based on other projects desires to reduce the number of configuration files for a project.

An additional (currently unused) reason is related to comparison reports which take into account the dependencies/devDependencies/peerDependencies of a project. This helps `filesize` give users a warning that a specific delta in size is mostly driven by a known large increase from a dependency.

Reply:
> We can add complexity into `filesize` for this case, but this preference was based on other projects desires to reduce the number of configuration files for a project.

Can we have our cake and eat it too, similar to Renovate, which supports [multiple config options](https://docs.renovatebot.com/configuration-options/)?

I suppose this comes down to a choice between the convenience of a separate file with separate owners, and the potential functionality of easily inspecting `package.json` for other dependency info. I'd argue that the former is a good thing to have soon, while the latter can still be done by separately reading `package.json`.

I'm resolving this comment thread as something to consider later on, and approving the PR.

Reply:
Sounds like an interesting option. I'll file a ticket on `filesize`.

Reply:
Just filed https://github.com/ampproject/filesize/issues/83